clinica/iotools/converters/aibl_to_bids/aibl_utils.py

```diff
@@ -235,17 +235,17 @@ def dicom_to_nii(subject, output_path, output_filename, image_path):
 
 
 def viscode_to_session(viscode):
-    """Replace the session label 'bl' with 'M00' or capitalize the session name passed as input.
+    """Replace the session label 'bl' with 'M000' or capitalize the session name passed as input.
 
     :param viscode: session name
 
-    :return: M00 if is the baseline session or the original session name
+    :return: M000 if is the baseline session or the original session name
     capitalized
     """
     if viscode == "bl":
-        return "M00"
+        return "M000"
```

Comment:
This function is exactly the same as the one in `adni_to_bids`.
Couldn't we import the former or have just one copy in a common utils module?

Reply:
I guess we can either move these 'common' functions to `converter_utils.py` or import the function from adni_utils.py.
WDYT ?